source/navigation/types.tsx

```diff
@@ -16,7 +16,7 @@ export type RootStackParamList = {
 	BuildingHoursDetail: {building: BuildingType}
 	BuildingHours: undefined
 	BuildingHoursProblemReport: {initialBuilding: BuildingType}
-	BuildingHoursScheduleEditor: HoursEditorProps
+	BuildingHoursScheduleEditor: HoursEditorProps['route']['params']
```

Comment:
Maybe there's a different way for us to define, import, and export these in a cleaner way?

Reply:
maybe make a `export type RouteParams = {xyz}; type Props { route: { params: RouteParams } };`?